x/subscription/keeper/subscription_test.go

```diff
@@ -518,9 +582,12 @@ func TestAddProjectToSubscription(t *testing.T) {
 
 	defaultProjectName := projectstypes.ADMIN_PROJECT_NAME
 	longProjectName := strings.Repeat(defaultProjectName, projectstypes.MAX_PROJECT_NAME_LEN)
+	projectNameWithComma := "projectName,"
```

Comment:
enough to have one test with disallowed character?
(tests for checking different disallowed characters should go with the helper that checks names/descriptions)

Reply:
see answer to same comment in project_test

Reply:
done